notebooks/03_global_forecasting.ipynb

```diff
@@ -20,9 +20,7 @@
     "* Foundation Models\n",
```

Comment:
<p>pretrained huggingface models </p><p> </p><p> </p><p>I would also note that these models are pretrained on very specific datasets (e.g. electrical, monthly tourism, or traffic time series)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-euroscipy2024/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>